lib/src/ticket/ticket_page.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+import '../../l10n/tr.dart';
+import 'cubits/ticket_cubit.dart';
+import 'picker.dart';
+
+class TicketPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        const Icon(
+          Icons.qr_code,
+          size: 200,
+        ),
+        const SizedBox(
+          height: 16,
```

Comment:
Only `SizedBox` should be without trailing commas when it's used for spacing like this.